src/Forms/HTMLEditor/TinyMCEConfig.php

```diff
@@ -844,4 +845,19 @@ protected function getAdminModule()
         Deprecation::notice('5.0', 'Set base_dir or editor_css config instead');
         return ModuleLoader::getModule('silverstripe/admin');
     }
+
+
+    /**
+     * Sets the upload folder name used by the insert media dialog
+     *
+     * @param string $folderName
+     * @return $this
+     */
+    public function setFolderName($folderName)
```

Comment:
Since it's going into `4`, you can type hint this.

Reply:
I thought I did type hint. What form did you want me to use?

Reply:
Did you mean `public function setFolderName($folderName) : TinyMCEConfig` ?